docs/UsersGuide/source/Graphics.rst

```diff
@@ -82,6 +84,8 @@ On Cheyenne:
 
 On Hera and Jet:
 
+NOTE DOM 20220227  - This /apps directory doesn't exist on jet (haven't checked hera)
+
 .. code-block:: console
 
```

Comment:
The /apps directory is only for Orion (Jet and Hera use /contrib).   The Gaea block can be changed to:

module use /lustre/f2/pdata/esrl/gsd/contrib/modulefiles
module load rocoto/1.3.3
module load miniconda3/4.8.3-regional-workflow

Reply:
Please check this Gaea information one more time. This is what is set for the workflow, but the python requirements for the plotting scripts are a bit different. For example, not sure why we would need module load rocoto/1.3.3 here.